File: dataset/human/java/file_9426.java
import java.util.*;

class Main{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    int s=sc.nextInt();
    int d1=0,d2=0,count=0;
    for(int i=0;i<s;i++){
      d1=sc.nextInt();
      d2=sc.nextInt();
      if(d1==d2){
        count++;
        if(count==3)break;
      }
      else{
        count=0;
      }
      
    }
    if(count>=3)System.out.println("Yes");
    else System.out.println("No");
  }
}

File: dataset/human/c/file_9426.c
#include <stdio.h>

int main(void)
{
   int n,d,x,a[100],i,j,count=0;

   scanf("%d",&n);
   scanf("%d%d",&d,&x);

   for(i=1;i<=n;i++)
      scanf("%d",&a[i]);

   for(i=1;i<=n;i++){
      for(j=1;j<=d;j=j+a[i]) count++;
   }
   printf("%d",count+x);

   return 0;
}
